tests/plotting/test_plotting.py

```diff
@@ -328,6 +328,17 @@ def test_plot_return_cpos(sphere):
     assert sphere.plot(return_cpos=False) is None
 
 
+def test_plot_return_img_without_cpos(sphere: pyvista.PolyData):
+    img = sphere.plot(return_cpos=False, return_img=True, screenshot=True)
+    assert isinstance(img, np.ndarray)
+
+
+def test_plot_return_img_with_cpos(sphere: pyvista.PolyData):
+    cpos, img = sphere.plot(return_cpos=True, return_img=True, screenshot=True)
+    assert isinstance(cpos, pyvista.CameraPosition)
+    assert isinstance(img, np.ndarray)
```

Comment:
As @MatthewFlamm pointed out, our image regression testing is currently broken, but these tests will likely need to add images for the cache as I believe the default is for all tests in `test_plotting.py` to use the cache... I think...

Reply:
See PR comment about moving to `tests/test_plotter.py`

Reply:
Sounds better. I will move them asap.

Reply:
Done